wavefront-hpa-adapter/values.yaml

```diff
@@ -0,0 +1,28 @@
+image:
+  repository: wavefronthq/wavefront-hpa-adapter
+  tag: 0.9.3
+  pullPolicy: IfNotPresent
+
+apiserver:
+  # Wavefront url of the form https://INSTANCE.wavefront.com
```

Comment:
Can you please pull the url and token under a top level `wavefront` property similar to the collector helm:
https://github.com/wavefrontHQ/helm/blob/868a9cea0810f0505a4e7a264b0409599f174570/wavefront/values.yaml#L11

It'd be nice for this to be consistent across all our helm charts.